lib/client.js

```diff
@@ -374,7 +389,7 @@ Client.prototype = {
       throw new Error('Invoke with an object isn\'t supported.');
     }
 
-    return wrappedPostMessage.call(this, 'invoke', obj);
+    return wrappedPostMessage.call(this, 'invoke', obj).then(throwUnlessBulk.bind(null, key));
```

Comment:
Invoke never has bulk operations does it? Can't we continue to use the existing .error property?